lib/core/macros.nim

```diff
@@ -87,17 +87,20 @@ type
     nnkFuncDef,
     nnkTupleConstr,
     nnkError,  ## erroneous AST node
+    nnkModuleRef
+    nnkReplayAction
+    nnkNilRodNode
```

Comment:
These have been left out deliberately. They are ROD file specific.

Reply:
Well i need a slot for these in case further enum members are added after that, or, we should document that new enum members go before `nnkModuleRef`

Reply:
Yup, new enum members go before `nnkModuleRef`

Reply:
fixed